.github/workflows/ci-test-ginkgo.yml

```diff
@@ -17,6 +17,7 @@ on:
       - "tests/**"
       - "protobuf/**"
       - ".github/workflows/ci-test-ginkgo.yml"
+      - "examples/multiubuntu/build/**"
```

Comment:
we'll need handling to use locally built image with added changes, but let's handle it later with a separate PR.

Reply:
Valid, let's do it as part of #1878